python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Commands.xml

```diff
@@ -91,10 +91,10 @@
   <SALCommand>
     <Subsystem>Electrometer</Subsystem>
     <EFDB_Topic>Electrometer_command_startScanDt</EFDB_Topic>
-    <Description>Start storing readings inside the electrometer buffer for a fixed time, this command will change detailedState from NotReadingState to SetDurationReadingState and to stop storing data into the buffer and publish the LFO there are 2 possibilities. Reading during scanDuration time input in the command or sending a stopScan command</Description>
+    <Description>Start storing readings inside the electrometer buffer for a fixed time given by scanDuration. This command will change detailedState from NotReadingState to SetDurationReadingState. It will stop storing data into the buffer and publish the LFA event. If no scanDuration is given, the buffer will stop storing once stopScan is sent.</Description>
```

Comment:
If no scanDuration is given, the buffer will stop storing once stopScan is sent.  --> Is that true? The code makes it appear that it would error out...

Reply:
startScanDt does fault when no duration is given. However, it also saves a file.

As for the ZeroCalib, we should add this automatically when mode or range are changed. This will be done in [DM-38329 -](https://jira.lsstcorp.org/browse/DM-38329)